docs/reference/cli.md

```diff
@@ -44,314 +47,433 @@ The line `Telemetry enabled` disappears when you disable it.
 `quickwit --version` displays the version. It is useful for reporting bugs.
 
 
+### Syntax
 
-### New
+The cli is structured into highlevel commands with subcommands.
+`quickwit [command] [subcommand] [args]`.
 
-*Description*
+* `command`:index, split, service, source. 
 
-Creates an index at `index-uri` configured by a json file located at `index-config-path`. The command fails if an index already exists at `index-uri` unless `overwrite` is passed. When `overwrite` is enabled, the command deletes all the files stored at `index-uri` before creating a new index. The index config defines how a document and fields it contains, are stored and indexed, see the [index config documentation](index-config.md).
 
-*Synopsis*
 
-```bash
-quickwit new
-    --index-uri <uri>
-    --index-config-path <path>
-    [--overwrite]
-```
 
-*Options*
-
-`--index-uri` (string) Defines the index location.<br />
-`--index-config-path` (string) Defines the index config path.<br />
-`--overwrite` (boolean) Overwrites existing index.
 
-*Examples*
 
-*Creating a new index on local file system*
+# index
+Index operation commands such as `create`, `index`, or `search`...
 
-```bash
-quickwit new --index-uri file:///quickwit-indexes/catalog --index-config-path ~/quickwit-conf/index_config.json
-```
+### index create
 
-Creating a new index on Amazon S3*
+Creates an index at `index-uri` configured by a yaml file located at `index-config-uri`. The command fails if an index already exists at `index-uri` unless `overwrite` is passed. 
```

Comment:
I don't think this sentence is accurate: `Creates an index at index-uri` 

This was the case for 0.1 but now the reference is the index ID. Index URI can be passed as an optional args.

Reply:
yeah it's more complicated now, I update the docs